src/lib/services/validator.ts

```diff
@@ -58,3 +60,29 @@ export const getValidators = async (
     {}
   );
 };
+
+export const resolveValIdentity = async (
+  apiPath: Option<string>,
+  validator: ValidatorInfo
+): Promise<string> => {
+  const githubUrl = `https://raw.githubusercontent.com/cosmostation/chainlist/master/chain/${apiPath}/moniker/${validator.validatorAddress}.png`;
+  const keybaseUrl = `https://keybase.io/_/api/1.0/user/lookup.json?key_suffix=${validator.identity}&fields=pictures`;
+  const uiAvatarsUrl = `https://ui-avatars.com/api/?name=${removeSpecialChars(
+    validator.moniker ?? ""
+  )}&background=${CURR_THEME.colors.secondary.main.replace("#", "")}&color=fff`;
```

Comment:
could we possibly create the fallback logo with code and use it as a fallback component for validator image? this could be useful for local too